### DISCUSSION TITLE
Trying to figure out when trains are called

### DISCUSSION THREAD

**Discussion by Evelynn20** [Opinionated-Blueprints/10-Books-Full-of-Rails#39] [Q&A] [answered]
I've built a basic system with an inline provider and inline requester with group [signal 6] and item [iron ore]. I've also built a 2-4-0 train with group [signal 6] and made depot and refuelling site for it. I was expecting it to try and refuel and then load itself at the refuelling station and the inline provider, but it seems intent on just waiting at the depot with only 50 solid fuel in the 2 locomotives. Am I missing a step? Using the V2.15.0 book

**Comment by Evelynn20** [comment]
Now looking at the train's schedule, it seems to want to go to the provider, but instantly switches back to the depot, back and forth so often it never moves.

**Comment by Bocian-1** [comment, marked as answer]
Did you check this?
https://github.com/Opinionated-Blueprints/10-Books-Full-of-Rails/wiki/Provider-and-Requester-stations
You also need a fuel supply station and train to supply the refuelling station. Otherwise, it will stay closed due to lack of fuel.

**Reply by Evelynn20** [reply]
Oh, didn't realise a separate train and fuelling station would be required for the refuelling station, thanks. I was more curious about how the system actually makes a call to the trains to arrive at the station.

**Reply by Bocian-1** [reply]
read the wiki and other discussion please, this is asked almost daily across github and factorioprints
